depends/packages/qt.mk

```diff
@@ -94,10 +90,10 @@ $(package)_config_opts += -no-feature-printdialog
 $(package)_config_opts += -no-feature-printer
 $(package)_config_opts += -no-feature-printpreviewdialog
 $(package)_config_opts += -no-feature-printpreviewwidget
-$(package)_config_opts += -no-feature-regularexpression
```

Comment:
@fanquake 
Was it done intentionally? If so, what reasons for?

Asking in regards to https://github.com/bitcoin-core/gui/pull/337/files#r636353521.

Reply:
From what I remember qt wouldn't compiler with this disabled.

Reply:
Thanks. Can confirm it.